python/ibmos2spark/osconfig.py

```diff
@@ -170,37 +170,50 @@ def url(self, container_name, object_name):
 
 class CloudObjectStorage(object):
```

Comment:
I think how it is right now is quite clean and clear. Honestly, i think having a new class for Bluemix COS would be confusing since it is really the same with different creds.